### ISSUE TITLE
Doesn't understand bundle/ folder (pathogen)

### ISSUE THREAD

**Issue by idbrii** [bulletmark/vim-plugins-profiler#1]
I tried vim-plugins-profiler and it gave me some unhelpful output:

```
$ pip3 install vim-plugins-profiler
$ vim-plugins-profiler
   1:    99.562 (94.3%) bundle
   2:     3.392 ( 3.2%) plugin
   3:     2.249 ( 2.1%) after
   4:     0.431 ( 0.4%) ftdetect
```

I use vim-pathogen to setup my plugins so they're all git clones to ~/.vim/bundle (each plugin has their own directory like ~/.vim/bundle/surround/). Seems unhelpful that it doesn't break down bundle into subsections.

**Comment by bulletmark** [comment]
It's extremely difficult to cater for all the different plug-in managers and configurations. Can you please provide the output of `ls -l ~/.vim`

**Comment by bulletmark** [comment]
Also, check the contents of your "plugin" dir. What is in there? If that is an old dir then delete it, update to current master version, and re-run.

**Comment by idbrii** [comment]
I don't want to waste your time, so I should say that I've since found [vim-plugins-profile](https://github.com/hyiltiz/vim-plugins-profile) which accomplishes a similar goal. I'll close this, but leave the information you requested if you still want to fix:

[This is my .vim](https://github.com/idbrii/daveconfig/tree/master/multi/vim).

Lots of folders. I also have a bunch of .vim files in ~/.vim/plugin/, but no subfolders (these are small plugins I've written but haven't promoted to their own repo). But the bulk of my plugins are in ~/.vim/bundle/

I understand the difficulty of making everyone happy, but I reported because I thought ~/.vim/bundle/ was very common. I could be wrong.

Thanks for trying to fix! I downloaded [vim_plugins_profiler.py](https://github.com/bulletmark/vim-plugins-profiler/blob/31c466be013af7eaf8c853f831c4f5aee9889b2b/vim_plugins_profiler.py) and ran it and got no output. (This is on a different machine than my original report -- now I'm on Windows10 but original was macOS).

If I add some logging:
```
print(paths)
paths = {p: v for p, v in paths.items() if
        len(p.parts) > 1 and p.parts[0] not in EXCLUDED}
print(paths)
```
I get this output:

```
defaultdict(<class 'float'>, {})
{}
defaultdict(<class 'float'>, {})
{}
defaultdict(<class 'float'>, {})
{}
defaultdict(<class 'float'>, {})
{}
```
